.github/workflows/saltstack.yml

```diff
@@ -1,11 +1,13 @@
+# vim: ft=yaml
+
 name: 'Saltstack'
 
 on: [push, pull_request]
 
 jobs:
   saltstack:
     name: 'Saltstack'
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-22.04
```

Comment:
You need to also adapt the `salt-minion` installation to the new Ubuntu version.

Reply:
Same for inspec